samples/pet/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+cmake_minimum_required(VERSION 3.13.3)
+project (sample_pet)
+
+# Generate encoding and decoding code for the 'Pet' structure.
+set(py_command_pet
+  zcbor code -d -e -b64 --short-names
```

Comment:
given that this is an example meant to serve as documentation for novice users I would recommend having these arguments on separate lines and adding a comment to what they mean and why they are set they way they are.